src/bundle/Resources/public/scss/_custom-url-form.scss

```
@@ -31,6 +32,11 @@
                 display: none;
             }
         }
+
+        svg {
```

```suggestion
        .ibexa-icon {
```
It is good to avoid styling by tags.